file-formats/eeec/type/zif_aff_eeec_v1.intf.abap

```diff
@@ -0,0 +1,42 @@
+"! <p class="shorttext synchronized" lang="en">Event Consumption Model</p>
+INTERFACE zif_aff_eeec_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Type</p>
+    ty_type TYPE string.
+
+  TYPES:
+    "! <p class="shorttext synchronized" lang="en">Event Type</p>
+    BEGIN OF ty_s_event_type,
+      "! <p class="shorttext synchronized" lang="en">Type</p>
+      "! Type
+      type  TYPE ty_type,
+      "! <p class="shorttext synchronized" lang="en">Entity Name</p>
+      "! Entity name
+      entity_name TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_s_event_type,
```

Comment:
Do we need `$required` fields here?

Reply:
I will check with Raphael

Reply:
'type' and 'entity_name' are required, now